se/commands/compare_versions.py

```diff
@@ -64,10 +63,7 @@ def compare_versions(plain_output: bool) -> int:
 
 			with tempfile.TemporaryDirectory() as work_directory_name:
 				# Copy the Git repo to a temp folder, so we can stash and pop with impunity.
-				# If we work directly on the real repo, ctrl + c may leave it in a stashed state unexpectedly.
-				# We have to use this function instead of shutil.copytree because shutil.copytree
-				# raises an error if the directory exists, in Python 3.6. Python 3.8+ has an option to ignore that.
-				copy_tree(target, work_directory_name)
+				shutil.copytree(target, work_directory_name)
```

Comment:
But, you have to add the option to ignore the error if the directory exists

Reply:
Pushed a change. Please take another look